docs/getting-started/install-elastic-endpoint.asciidoc

```diff
@@ -54,7 +54,7 @@ image::images/install-endpoint/filter-network-content.png[]
 [[enable-fda-endpoint]]
 == Enable Full Disk Access for {elastic-endpoint}
 
-{elastic-endpoint} requires Full Disk Access to subscribe to system events via the {elastic-defend} framework and to protect your network from malware and other cybersecurity threats. To enable Full Disk Access on endpoints running macOS Catalina (10.15) and later, you must manually approve {elastic-endpoint}. For endpoints running macOS Mojave (10.14) and earlier, you must also approve the {elastic-endpoint} <<kernel-extension-approval, kernel extension>>.
+{elastic-endpoint} requires Full Disk Access to subscribe to system events via the {elastic-defend} framework and to protect your network from malware and other cybersecurity threats. To enable Full Disk Access on endpoints running macOS Catalina (10.15) and later, you must manually approve {elastic-endpoint}. 
 
 NOTE: The following instructions apply only to {elastic-endpoint} running {stack} version 8.0.0 and later. To see Full Disk Access requirements for the Endgame sensor, refer to <<endgame-sensor-full-disk-access>>.
```

Comment:
I know this is out of scope for this PR, but I happened to notice this line: I don't think it's accurate to say "Elastic Endpoint running Stack version X.y.z and later," because Endpoint doesn't run the Stack. Maybe something simpler like "Elastic Endpoint version X.y.z and later"?

Reply:
Oh yeah, good catch. I think we're good to remove `stack` from here and just stick with "Elastic Endpoint version x.y.z and later". I'll make this change before merging.

Reply:
Addressed in [a1c1d61](https://github.com/elastic/security-docs/pull/3564/commits/a1c1d61ecbb79cff293f5d78692e52b80e4cff99).